typings/styled-components.d.ts

```diff
@@ -36,148 +60,23 @@ export type HtmlStyledFunction<E> = ThemedHtmlStyledFunction<E, any>;
 export type ThemedSvgStyledFunction<E extends SVGElement, T> = ThemedStyledFunction<React.SVGAttributes<E>, T>;
 export type SvgStyledFunction<E extends SVGElement> = ThemedSvgStyledFunction<E, any>;
 
-export interface ThemedBaseStyledInterface<T> {
+type ThemedStyledComponentFactoriesHTML<T> = {
```

Comment:
Minor: would Pascal-case `Html`, `Svg` be better for consistency (`ThemedShtmlStyledFunction`)